pkg/infra/log/log.go

```diff
@@ -99,20 +177,87 @@ func getFilters(filterStrArray []string) map[string]log15.Lvl {
 	return filterMap
 }
 
-func getLogFormat(format string) log15.Format {
+func Stack(skip int) string {
+	call := stack.Caller(skip)
+	s := stack.Trace().TrimBelow(call).TrimRuntime()
+	return s.String()
+}
+
+type Formatedlogger func(w io.Writer) gokitlog.Logger
+
+func terminalColorFn(keyvals ...interface{}) term.FgBgColor {
+	for i := 0; i < len(keyvals)-1; i += 2 {
+		if keyvals[i] != level.Key() {
+			continue
+		}
+		switch keyvals[i+1] {
+		case "trace":
+			return term.FgBgColor{Fg: term.DarkGray}
+		case "debug":
+			return term.FgBgColor{Fg: term.DarkGray}
+		case "info":
+			return term.FgBgColor{Fg: term.Gray}
+		case "warn":
+			return term.FgBgColor{Fg: term.Yellow}
+		case "error":
+			return term.FgBgColor{Fg: term.Red}
+		case "crit":
+			return term.FgBgColor{Fg: term.Gray, Bg: term.DarkRed}
+		default:
+			return term.FgBgColor{}
+		}
+	}
+	return term.FgBgColor{}
+}
+
+func getLogFormat(format string) Formatedlogger {
 	switch format {
 	case "console":
 		if isatty.IsTerminal(os.Stdout.Fd()) {
-			return log15.TerminalFormat()
+			return func(w io.Writer) gokitlog.Logger {
+				return term.NewLogger(w, gokitlog.NewLogfmtLogger, terminalColorFn)
+			}
```

Comment:
```suggestion
                        if strings.ToLower(runtime.GOOS) != "windows" {
			    return func(w io.Writer) gokitlog.Logger {
				    return term.NewLogger(w, gokitlog.NewLogfmtLogger, terminalColorFn)
			    }
                        }
```

As alternative to not break stdout in Windows. It will fallback to a regular logfmt logger